<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DifficultyVarPage</class>
 <widget class="QWidget" name="DifficultyVarPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="Vulnerability">
     <property name="text">
      <string>Vulnerability</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="RealisticPilotVulnerability">
     <property name="text">
      <string>Realistic pilot vulnerability</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="HeadShake">
     <property name="text">
      <string>Head shake</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="WindTurbulence">
     <property name="text">
      <string>Wind &amp;&amp; turbulence</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="Clouds">
     <property name="text">
      <string>Clouds</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="NoInstantSuccess">
     <property name="text">
      <string>No instant success</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="TakeoffLanding">
     <property name="text">
      <string>Takeoff &amp;&amp; landing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="RealisticLanding">
     <property name="text">
      <string>Realistic landing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="RealisticNavigationInstruments">
     <property name="text">
      <string>Realistic navigation instruments</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
